chart/epinio-ui/templates/server.yaml

```diff
@@ -38,11 +38,11 @@ spec:
 
         env:
         - name: EPINIO_API_URL
-          value: {{ default (printf "http://epinio-server.%s.svc.cluster.local" .Release.Namespace) .Values.epinioApiUrl }}
+          value: {{ default (printf "http://epinio-server.%s.svc.cluster.local" .Release.Namespace) .Values.epinioAPIUrl }}
```

Comment:
Are we following [golang conventions](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)? In that case, it should be `epinioAPIURL` or `epinioAPIurl` (I prefer the first).

Reply:
Sure, lets go full golang

Reply:
Done. Commit 750341131b59f564ad6eba925a24508a5784d7b8